test/integration/audit_trail_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+class AuditTrailTest < ActionDispatch::IntegrationTest
+  def setup
+    @user = create :user, email: "first_user@email.com"
+    @user2 = create :user, email: "second_user@email.com"
+    log_in_as @user
+    # making stuff happen in the backend for now
+    @patient = Patient.create!(name: "tester", primary_phone: "1231231234")
+    @pregnancy = create :pregnancy, appointment_date: nil, patient: @patient
+    visit edit_pregnancy_path(@pregnancy)
+  end
+
+  describe 'A patient was created at some point' do
+    it 'posts changes and verifies audit trail in db' do
+      assert_equal @patient.history_tracks.count , 1
+      assert_equal @patient.created_by, @user
+      assert_equal page.find(:xpath, '//*[@id="pregnancy_patient_name"]').value , "tester"
```

Comment:
I usually use `page.find(:id, 'id_string')` to avoid trying to remember xpath exists, haha. 

I feel like some day I will be describing trying to use xpath at my first programming job to a therapist.